src/main/webapp/app/tasks/local-task.service.ts

```diff
@@ -32,6 +35,24 @@ export class LocalTaskService implements TaskService {
     return of(null);
   }
 
+  done(id: string): Observable<void> {
+    const tasks = this.readTasks();
+    const index = tasks.findIndex(task => task.id === id);
+    tasks[index].isComplete = !tasks[index].isComplete;
+    tasks.sort((a, b) => {
+      return Number(a.isComplete) - Number(b.isComplete);
```

Comment:
This arrow function could be shortened by omitting the curly braces and the return keyword.